cmd/lakectl/cmd/ingest.go

```diff
@@ -12,6 +14,32 @@ const ingestSummaryTemplate = `
 Staged {{ .Objects | yellow }} external objects (total of {{ .Bytes | human_bytes | yellow }})
 `
 
+type stageRequest struct {
+	ctx        context.Context
```

Comment:
ctx can be passed to the workers once on creation.

Reply:
Done